### ISSUE TITLE
List of Commands

### ISSUE THREAD

**Issue by gaaamii** [gaaamii/wdim#4]
On my Ubuntu, got the commands by:

```
$ ls /bin
```

bash
bunzip2
busybox
bzcat
bzcmp
bzdiff
bzegrep
bzexe
bzfgrep
bzgrep
bzip2
bzip2recover
bzless
bzmore
cat
cgroups-mount
cgroups-umount
chgrp
chmod
chown
chvt
cp
cpio
dash
date
dbus-cleanup-sockets
dbus-daemon
dbus-uuidgen
dd
df
dir
dmesg
dnsdomainname
domainname
dumpkeys
echo
ed
egrep
false
fgconsole
fgrep
findmnt
fuser
fusermount
grep
gunzip
gzexe
gzip
hostname
init-checkconf
initctl2dot
ip
kbd_mode
kill
less
lessecho
lessfile
lesskey
lesspipe
ln
loadkeys
login
lowntfs-3g
ls
lsblk
lsmod
mkdir
mknod
mktemp
more
mount
mountpoint
mt
mt-gnu
mv
nano
nc
nc.openbsd
netcat
netstat
nisdomainname
ntfs-3g
ntfs-3g.probe
ntfs-3g.secaudit
ntfs-3g.usermap
ntfscat
ntfsck
ntfscluster
ntfscmp
ntfsdecrypt
ntfsdump_logfile
ntfsfix
ntfsinfo
ntfsls
ntfsmftalloc
ntfsmove
ntfstruncate
ntfswipe
open
openvt
pidof
ping
ping6
plymouth
plymouth-upstart-bridge
ps
pwd
rbash
readlink
rm
rmdir
rnano
running-in-container
run-parts
sed
setfont
setupcon
sh
sh.distrib
sleep
ss
static-sh
stty
su
sync
tailf
tar
tempfile
touch
true
ulockmgr_server
umount
uname
uncompress
unicode_start
vdir
which
whiptail
ypdomainname
zcat
zcmp
zdiff
zegrep
zfgrep
zforce
zgrep
zless
zmore
znew

**Comment by saildata** [comment]
I came across your project this evening before starting something similar.. Regarding your to-do list, would it make sense to write a quick script to loop over the relevant man pages and grab the line from the name section? Or just pipe it through `man -k`? It would still have to be cleaned before production but I think that would expedite the outstanding terms on your list. Again, good idea - you beat me to it : )

**Comment by gaaamii** [comment]
@saildata 

Yes. I think this list is not sufficient for our wants. And I think your idea is good. Also, If you want more terms for wdim, please add those terms without hesitation.

Honestly, I don't care this repository currently. But you encouraged me to add more terms to wdim. I'm so excited that you are interested in wdim.

Thanks for a good advise 👍

**Comment by saildata** [comment]
@gaaamii  Thank you for the quick response! x 💯 

You never know on projects without active development whether the person is even still interested. 

I think this would be fairly easy to hack together. I wrote the following just now as an example and while it wouldn't gain 1000 pts on Stack Overflow, it gets the job done, using ntfs-3g as the search term ( ❗ TODO: loop through input list or other way to grab your remaining items)

``` shell
man -k find ntfs-3g | grep ntfs-3g | sed 's/ - / /' | cut -d' ' -f1,3-
```

Quick explaination:  
- Use man (apropos) to search
- It returns too many results, so narrow it down with grep
- The dashes make a difficult split, so remove them with the sed utility
- Finally, grab the first column (the name), and the third through the end of the line 
- Profit!

Output (Arch Linux)  

``` shell
mount.lowntfs-3g Third Generation Read/Write NTFS Driver
mount.ntfs-3g    Third Generation Read/Write NTFS Driver
ntfs-3g          Third Generation Read/Write NTFS Driver
ntfs-3g.probe    Probe an NTFS volume mountability
ntfs-3g.secaudit NTFS Security Data Auditing
ntfs-3g.usermap  NTFS Building a User Mapping File
```

Let me know what you think --

**Comment by gaaamii** [comment]
I'm sorry for my late response. I think your idea is good. 
I'd like to use that to improve my `config/dictionary.yml`.
Thank you :)

**Comment by saildata** [comment]
> I'm sorry for my late response. I think your idea is good.
> I'd like to use that to improve my `config/dictionary.yml`.
> Thank you :)

Everyone is on a different timeline, no worries! I'm glad it is of some use to your project :slightly_smiling_face:
